wordpress-mobile/WordPress-iOS
# Post Epilogue: Pin navigation bar to top layout guide

Fixes #9691. This PR updates the PostPost storyboard to use autolayout constraints for the navigation bar, to pin it to the top layout guide and improve its positioning.

![simulator screen shot - iphone xr - 2019-01-07 at 11 24 35](https://user-images.githubusercontent.com/4780/50766585-b5ab0800-1271-11e9-854a-ce44ccf918a8.png)
![simulator screen shot - iphone xr - 2019-01-07 at 11 24 40](https://user-images.githubusercontent.com/4780/50766586-b8a5f880-1271-11e9-84cc-b3ac88cec67f.png)

**To test**

* Build and run on an X-series device / simulator
* Publish a post, and tap View in the notice that's displayed on successful publish
* Check that the Done button is not too close to the top of the screen. Rotate and check that it respects the safe area insets in landscape.

---

Update release notes:
